src/_guides/language/effective-dart/usage.md

```diff
@@ -21,6 +21,106 @@ library may not be able to tell that you've internalized the ideas here, but
 * TOC
 {:toc}
 
+## Libraries
+
+These guidelines help you compose your program out of multiple files in a
+consistent, maintainable way. To keep these guidelines brief, they use "import"
+to cover import and export directives. The guidelines apply equally to both.
+
+### DO use strings in part-of directives.
+
+When you use `part` to split part of a library out into a second file, that
+other file needs to in turn indicate which library it's a part of. For legacy
+reasons, Dart allows a `part of` directive to use the *name* of the library it's
+a part of. That makes it harder for tools to physically find the main library
+file, and can make it ambiguous which library the part is actually part of.
+
+The preferred, modern syntax is to use a URI string that points directly to the
+library file, just like you use in other directives. If you have some library,
+"my_library.dart", that contains:
+
+<?code-excerpt "misc/lib/effective_dart/my_library.dart (part)"?>
+{% prettify dart %}
+library my_library;
+
+part "some/other/file.dart";
+{% endprettify %}
+
+Then the part file should look like:
+
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/some/other/file.dart (part-of)"?>
+{% prettify dart %}
+part of "../../my_library.dart";
+{% endprettify %}
+
+And not:
+
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/some/other/file_2.dart (part-of)"?>
+{% prettify dart %}
+part of my_library;
+{% endprettify %}
+
+### DON'T import libraries inside the "src" directory of another package.
+
+The "src" directory under "lib" is specified to contain libraries private to the
```

Comment:
Google style guidelines [recommend](https://developers.google.com/style/code-in-text) that folder names be written in `code` font w/o quotes.

The guideline isn't clear about file/folder names in headings, but I suppose that it holds there too.

Reply:
You're right about the `code` font for directory names. I usually avoid code font in headings (where font changes just look bad), so I don't mind the quotation marks there.